docs/release_notes-5.1/master.adoc

```diff
@@ -0,0 +1,20 @@
+:toc:
+:toclevels: 4
+:numbered:
+
+include::topics_5/templates/document-attributes.adoc[]
+
+:imagesdir: topics_5/images
+:context: release-notes
+:release-notes:
+
+= {ReleaseNotesName}
+
+//Inclusive language statement
+include::topics_5/making-open-source-more-inclusive.adoc[]
+
+These release notes describe the new features and changes implemented for the {ProductName}{nbsp}({ProductShortName}){nbsp}{DocInfoProductNumber} release.
```

Comment:
I mentioned in an earlier comment that this should be moved above the inclusive language include. Otherwise they will get lost.

I also think you should use the z-stream version here.